ExpressionUtils/ExpressionUtils.csproj

```diff
@@ -19,6 +19,8 @@
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageProjectUrl>https://github.com/Miaplaza/expression-utils</PackageProjectUrl>
     <PackageTags>Expression;Expressions;Linq.Expressions;Evaluation;Compile;Compiler;Execute;Execution;Compare;Comparison</PackageTags>
+    <DebugType>embedded</DebugType>
+    <EmbedAllSources>true</EmbedAllSources>
```

Comment:
Is this intended to be merged? It seems in the other projects we rather use the default `portable` setting https://miaplaza.slack.com/archives/CMFMGHQ05/p1591812385012600

Reply:
It is intended. So for some reason i couldn't add breakpoints to places inside ExpressionUtills when debugging the website. Leo recommended to add this to resolve this issue, as we do that in other projects  (for example regulator). And it actually helped and i was able to add breakpoints. Tried to figure out why it helped but was not able to. All i know with this change it works. Since it is a public repo including sources should be safe. 

<img width="580" alt="{59E12CDE-17F4-4AB0-9A00-5054684CC246}" src="https://github.com/user-attachments/assets/5ee0b308-053c-40b8-addf-4c5c60d86dcb">